airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/kotlin/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.kt

```diff
@@ -58,7 +58,7 @@ abstract class BaseSqlGeneratorIntegrationTest<DestinationState : MinimumDestina
     protected var cdcIncrementalAppendStream: StreamConfig? = null
 
     protected var generator: SqlGenerator? = null
-    protected abstract var destinationHandler: DestinationHandler<DestinationState>
+    protected abstract val destinationHandler: DestinationHandler<DestinationState>
```

Comment:
not sure why it was defined as a var... Much better as a val. How did you find it?

Reply:
making var generated abstract `setDestinationHandler` method which was never intended to be, just `getDestinationHandler`